src/ngx-smart-loader/src/services/ngx-smart-loader.service.ts

```diff
@@ -184,4 +165,52 @@ export class NgxSmartLoaderService {
     }
   }
 
+  /**
+   * Get the loader from the loaderStack and start it
+   * If the loader is NOT created yet, we store the start action as a promise to execute it at the loader creation
+   *
+   * @param identifier The loader identifier.
+   * */
+  private doStart(identifier): void {
+    const loader = this.getLoader(identifier);
+    // The loader is not created yet, I store the start action to execute it when the loader will be added
+    if (!loader) {
+      this.addWaitingAction(identifier, 'onStart', this.doStart);
+      return;
+    }
+
+    loader.start();
+  }
+
+  /**
+   * Get the loader from the loaderStack and stop it
+   * If the loader is NOT created yet, we store the stop action as a promise to execute it at the loader creation
+   *
+   * @param identifier The loader identifier.
+   * */
+  private doStop(identifier): void {
```

Comment:
The method is private, it is necessary to add jsDoc comment?

Reply:
@mcsky Yes it is! It's not giving informations for public API but it can help for external developers who wants to contribute and/or maintain the project. 😉